helm-service/deploy/service.yaml

```diff
@@ -52,6 +52,11 @@ spec:
             configMapKeyRef:
               name: ingress-config
               key: ingress_port
+        - name: INGRESS_GATEWAY
```

Comment:
How much work would it be to rename `INGRESS_GATEWAY` into `ISTIO_GATEWAY`? Because this would follow the Istio naming https://istio.io/latest/docs/reference/config/networking/gateway/